src/vellum/workflows/events/tests/test_event.py

```diff
@@ -73,6 +73,7 @@ class MockWorkflow(BaseWorkflow[MockInputs, BaseState]):
                         "foo": "bar",
                     },
                 },
+                "parent": None,
```

Comment:
[blocking] let's add a test somewhere here to ensure we serialize parent context correctly

Reply:
🫡